ssv/spectest/tests/runner/pre_consensus_justifications/wrong_beacon_role.go

```diff
@@ -0,0 +1,111 @@
+package pre_consensus_justifications
+
+import (
+	"github.com/bloxapp/ssv-spec/qbft"
+	"github.com/bloxapp/ssv-spec/ssv/spectest/tests"
+	"github.com/bloxapp/ssv-spec/types"
+	"github.com/bloxapp/ssv-spec/types/testingutils"
+)
+
+// WrongBeaconRole tests wrong ConsensusData.Duty.Type != runner.BeaconRoleType
+func WrongBeaconRole() tests.SpecTest {
```

Comment:
can you comment on where in the code you feed the wrong type?

Reply:
done